vcd/resource_vcd_vapp_vm_with_vm_sizing_test.go

```diff
@@ -108,8 +111,8 @@ func TestAccVcdVAppVmWithVmSizing(t *testing.T) {
 
 					resource.TestCheckResourceAttrPair("vcd_vapp_vm."+netVmName1, "sizing_policy_id",
 						"vcd_vm_sizing_policy.size_cpu", "id"),
-					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "cpus", "3"),
-					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "cpu_cores", "3"),
+					resource.TestCheckResourceAttr("vcd_vapp_vm."+netVmName1, "cpus", "1"),
```

Comment:
I lowered the CPUs here simply because of limited resources in our testing environments.